app/src/main/java/com/hover/stax/transfers/TransferViewModel.kt

```diff
@@ -116,6 +119,50 @@ class TransferViewModel(application: Application, repo: DatabaseRepo) : Abstract
         }
     }
 
+    fun initNonTemplateVariables(entries: List<String>) {
+        val itemList = ArrayList<NonTemplateVariable>()
```

Comment:
Use a mapping function to initialise the `NonTemplateVariable` from the entries.

Reply:
Done, thanks. I like the map function. Cleaner.